Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1364,7 +1364,9 @@ PhysicalParticleContainer::Evolve (int lev,
         }
     }
     // Split particles
-    if (do_splitting){ SplitParticles(lev); }
+    if (do_splitting && a_dt_type == DtType::SecondHalf && WarpX::do_subcycling == 1){
```

Comment:
Should this be
```
    if (do_splitting && (a_dt_type == DtType::SecondHalf || WarpX::do_subcycling == 0) ){
```
Otherwise, it seems that particles will not be split, when we do not use subcycling.

Reply:
Also, could you add a comment here explaining why we can only split in the second half, in the case of subcycling.

Reply:
Another possibility could be:
```
if (do_splitting && (a_dt_type == DtType::SecondHalf || a_dt_type == DtType::Full) ){
```